scene/3d/portal.cpp

```diff
@@ -69,6 +70,40 @@ Portal::~Portal() {
 	}
 }
 
+String Portal::get_configuration_warning() const {
+	String warning = Spatial::get_configuration_warning();
+
+	auto lambda = [](const Node *p_node) {
+		if (Object::cast_to<RoomManager>(p_node) || Object::cast_to<Room>(p_node) || Object::cast_to<RoomGroup>(p_node)) {
+			return true;
+		}
+		return false;
```

Comment:
Could just be:
```suggestion
		return Object::cast_to<RoomManager>(p_node) || Object::cast_to<Room>(p_node) || Object::cast_to<RoomGroup>(p_node);
```

Reply:
Yes good point, will change tomorrow. :+1: